api/tokens.go

```diff
@@ -46,6 +47,26 @@ func (api *API) TokensHandler(ctx context.Context) http.HandlerFunc {
 		validEmailRequest := emailValidation(authParams)
 
 		if validPasswordRequest && validEmailRequest {
+
+			err = adminUserGlobalSignOut(authParams, api.UserPoolId, api.CognitoClient)
```

Comment:
Consider renaming this - something like - `terminateExistingSession` is a bit more user friendly and a bit more obvious about what its doing.

Reply:
Thank you, this is a good idea